php/classes/controllers/class-extensions-controller.php

```diff
@@ -113,13 +114,25 @@ public function render_seriously_simple_extensions() {
 				'new_window'  => true,
 				'button_text' => __( 'Get Second Line Themes', 'seriously-simple-podcasting' ),
 			),
+			'elementor-templates' => array(
+				'title'       => __( 'Elementor Templates', 'seriously-simple-podcasting' ),
+				'image'       => $image_dir . 'elementor.png',
+				'url'         => admin_url( 'edit.php?post_type=podcast&page=podcast_settings&tab=extensions&elementor-import-templates=true' ),
```

Comment:
We should create a nonce to be passed in the url, so that we can verify the nonce when the request runs

https://developer.wordpress.org/reference/functions/wp_create_nonce/